src/preload/client/documents.ts

```diff
@@ -1,14 +1,15 @@
 import { Database } from "better-sqlite3";
 import { Knex } from "knex";
 import { uuidv7 } from "uuidv7";
+import { IFilesClient } from "./files";
 
 export interface GetDocumentResponse {
   id: string;
   createdAt: string;
   updatedAt: string;
   title?: string;
   content: string;
-  journalId: string;
+  journal: string;
```

Comment:
Re-worked the journal references from id to name, to support reconciling synced journal names with what's in the cache. Probably this was a mistake, but I tied up the change too tightly to all the other work; may re-visit this in the near future.